examples/example.cpp

```diff
@@ -13,12 +13,11 @@ struct ExampleMiddleware
 {
     std::string message;
 
-    ExampleMiddleware() 
+    ExampleMiddleware() : message(std::string("foo"))
```

Comment:
`message("foo")`?

Reply:
originally this was defined the the body (2 lines below)

Reply:
earlier i tried codacy analysis with crow there was just 2-3 minor issues, instead of creating a separate PR for such minor things i thought to pick them along with this, that's why i changed this

Reply:
Yeah, I know. I had on mind change from `message(std::string("foo"))` to `message(."foo")`

Reply:
the example seems to work fine without the std::string section, seems unnecessary, could you remove it?

Reply:
Also line 86 still contains a request parameter, could you remove it?

Reply:
fixed both issues